pkg/motor/create_bucket.go

```diff
@@ -13,41 +12,41 @@ import (
 	bt "github.com/sonr-io/sonr/x/bucket/types"
 )
 
-func (mtr *motorNodeImpl) CreateBucket(ctx context.Context, request mt.CreateBucketRequest) (bucket.Bucket, error) {
```

Comment:
Will this change break speedway

Reply:
It'll just require changes next time speedway upgrades to this version

Reply:
Yea, speedway won't break if it doesn't update to it.